packages/lib-classifier/src/store/Subject.js

```diff
@@ -14,10 +16,31 @@ const Subject = types
     selected_at: types.maybe(types.string),
     selection_state: types.maybe(types.string),
     shouldDiscuss: types.maybe(types.frozen()),
-    user_has_finished_workflow: types.optional(types.boolean, false)
+    user_has_finished_workflow: types.optional(types.boolean, false),
+    transcriptionReductions: types.maybe(TranscriptionReductions)
   })
 
   .actions(self => {
+    function afterAttach () {
+      fetchTranscriptionReductions()
+    }
+
+    function beforeDestroy () {
+      self.transcriptionReductions && destroy(self.transcriptionReductions)
+    }
+
+    function fetchTranscriptionReductions () {
+      const subjectWorkflowDisposer = autorun(function subjectWorkflowDisposer () {
+        if (self.workflow && self.workflow.usesTranscriptionLines) {
+          self.transcriptionReductions = TranscriptionReductions.create({
+            caesarReducerKey: 'ext-17',
```

Comment:
What’s the best way to switch this between staging and production values?

Reply:
Cool. In that case, this can be defined as a constant on the `TranscriptionReductions` model.

Reply:
@CKrawczyk suggests that this value should be `alice` for the text line reducer.